src/w2field.js

```diff
@@ -1613,15 +1615,14 @@ class w2field extends w2base {
                     <input name="attachment" class="file-input" type="file" tabindex="-1"'
                         style="width: 100%; height: 100%; opacity: 0" title=""
                         ${this.options.max !== 1 ? 'multiple' : ''}
-                        ${query(this.el).prop('readonly') ? 'readonly': ''}
-                        ${query(this.el).prop('disabled') ? 'disabled': ''}
+                        ${query(this.el).prop('readOnly') || query(this.el).prop('disabled') ? 'disabled': ''}
```

Comment:
not sure why this is needed, but I think it would not do any harm. I rarely use file upload fields myself.

Reply:
I had just noticed that when I disabled file upload fields, the readonly field didn't stop functionality.